Tasks/AzureTestPlanV0/publishAutomatedTests.ts

```diff
@@ -0,0 +1,95 @@
+import tl = require('azure-pipelines-task-lib/task');
+import * as path from 'path';
+import constants = require('./constants');
+
+function publish(testRunner, resultFiles, mergeResults, failTaskOnFailedTests, platform, publishRunAttachments, testRunSystem , failTaskOnFailureToPublishResults) {
+    var properties = <{ [key: string]: string }>{};
+    properties['type'] = testRunner;
+
+    if (mergeResults) {
```

Comment:
what's the harm in storing default values for properties?

Reply:
Refactor to store values and have default since they most of them are Boolean with default state as false